src/qz/printer/status/CupsStatusHandler.java

```diff
@@ -64,6 +65,7 @@ private synchronized void getNotifications() {
                 int attrCount = cups.ippGetCount(jobStateReasonsAttr);
                 for (int i = 0;  i < attrCount; i++) {
                     String reason = cups.ippGetString(jobStateReasonsAttr, i, "");
+                    reason = statusFix(reason);
```

Comment:
Why not just do this in the map?

Reply:
I wanted to apply the fix as close to the cups side of things a possible. That way the status will already be changed for the duplicate detection.

Reply:
The map's job is to parse these reasons into enumerated types.  I seems best there, no (and it's only one line lower 😉 )